src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindBaseSymbolsCommandHandler.cs

```diff
@@ -66,31 +66,32 @@ protected override bool TryExecuteCommand(int caretPosition, Document document,
             {
                 using var token = _asyncListener.BeginAsyncOperation(nameof(StreamingFindBaseSymbolsAsync));
 
-                var context = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true, cancellationToken);
+                var (context, combinedCancellationToken) = presenter.StartSearch(EditorFeaturesResources.Navigating, supportsReferences: true, cancellationToken);
+                cancellationToken = combinedCancellationToken;
 
                 using (Logger.LogBlock(
                     FunctionId.CommandHandler_FindAllReference,
                     KeyValueLogMessage.Create(LogType.UserAction, m => m["type"] = "streaming"),
-                    context.CancellationToken))
+                    cancellationToken))
                 {
                     try
                     {
 #pragma warning disable CA2007 // Consider calling ConfigureAwait on the awaited task
-                        var relevantSymbol = await FindUsagesHelpers.GetRelevantSymbolAndProjectAtPositionAsync(document, caretPosition, context.CancellationToken);
+                        var relevantSymbol = await FindUsagesHelpers.GetRelevantSymbolAndProjectAtPositionAsync(document, caretPosition, cancellationToken);
 #pragma warning restore CA2007 // Consider calling ConfigureAwait on the awaited task
```

Comment:
i have no clue why none of the cocoa code avoids calling ConfigureAwait.  @KirillOsenkov do you know why?

Reply:
I have no idea. It was originally written like this by @avodovnik and then @tmat added the pragmas in a bulk commit to clean up warnings: https://github.com/dotnet/roslyn/commit/589da9de5ec3e2bd2a58d727c92ea76978ffaa0e

Reply:
ConfigureAwait is generally discouraged in JTF-aware code.

Reply:
so i don't think this code is JTF aware.  My preference would just be to remove the pragmas and `ConfigureAwait(false) ` all this code as it feels safe to do so.  Waiting on @KirillOsenkov though to determine if that's the case.